sgkit/io/vcf/vcf_reader.py

```diff
@@ -322,12 +324,13 @@ def vcf_to_zarr(
 ) -> None:
     """Convert VCF files to a single Zarr on-disk store.
 
-    For a single input and a single region, the conversion is carried out sequentially.
+    For a single input and a single region, the conversion is carried out sequentially,
+    unless `target_part_size` is specified.
 
-    For multiple outputs or regions, the conversion is carried out in parallel, by writing
-    the output for each region to a separate, intermediate Zarr store in ``tempdir``. Then,
-    in a second step the intermediate outputs are concatenated and rechunked into the final
-    output Zarr store in ``output``.
+    For multiple outputs or regions, or if `target_part_size` is specified, the conversion
```

Comment:
How are the results sorted?  Are there any guarantees for that or ways to ensure that the contig + genomic position  sort order is preserved?

Reply:
This is a good question and something worth documenting. Within each VCF file, order is preserved, since regions are numbered sequentially, and concatenated in sequence order. For multiple files, the order is the order that the files are specified in (by the user), and usually corresponds to the order that the contigs are arranged in.

Reply:
I see, nice.  Worth adding a test for it?

Reply:
I was just thinking that too 😄

Reply:
I've opened #425 to address this